torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -678,6 +678,141 @@ bool ProcessGroupNCCL::CoalescedWorkNCCL::wait(
 
 static std::atomic<size_t> process_group_id = 0;
 
+ProcessGroupNCCL::ProcessGroupNCCL(
```

Comment:
What is missing from the existing nccl constructor that DistributedBackendOptions is providing?

Reply:
The global ranks associated with each PG.